WTraining/test/users/post.spec.js

```diff
@@ -0,0 +1,121 @@
+const request = require('supertest');
+const { factory } = require('factory-girl');
+const app = require('../../app');
+const { factoryByModel } = require('../factory/factory_by_models');
+
+const ENDPOINT = '/users';
+
+let response = {};
+let userData = {};
+
+describe(`POST ${ENDPOINT}`, () => {
+  beforeAll(async done => {
+    await factoryByModel('User');
+    userData = await factory.build('User', { password: 'ABC123ab', email: 'asasas@wolox.com.ar' });
+    done();
```

Comment:
This done() is not necessary, do you know why? Check here https://jestjs.io/docs/asynchronous